viz.yaml

```diff
@@ -33,6 +33,24 @@ info:
     vizlab:
       repo: CRAN
       version: 0.2.0
+    sf:
+      repo: CRAN
+      version: 0.5-4
+    sp: 
```

Comment:
If we don't need sp for anything, I'd say we leave it out of the reusable function stuff.

Reply:
Need `sp` to use for geoknife::simplegeom